airflow/api_fastapi/app.py

```diff
@@ -78,20 +78,20 @@ def create_app(apps: str = "all") -> FastAPI:
         root_path=root_path,
     )
 
+    if "execution" in apps_list or "all" in apps_list:
+        task_exec_api_app = create_task_execution_api_app()
+        init_error_handlers(task_exec_api_app)
+        app.mount("/execution", task_exec_api_app)
```

Comment:
We needed to mount it earlier?

Reply:
Ah because of `test_catch_all_route_last` I'm guessing. 👍🏻

Reply:
Yep! It fails in not-totally-obvious ways if the catch all isn't last, so I figured a test to save our sanity was worthwhile.